ios/sdk/src/JitsiMeetView.m

```diff
@@ -98,7 +99,7 @@ - (instancetype)initWithFrame:(CGRect)frame {
  */
 - (void)initWithXXX {
```

Comment:
The `initWithXXX` is not too describing function name. Maybe `initWithExternalAPIScope `will be better.